proto/stat_var.proto

```diff
@@ -232,4 +232,22 @@ message GetStatVarSummaryRequest {
 }
 message GetStatVarSummaryResponse {
   map<string, StatVarSummary> stat_var_summary = 1;
+}
+
+message GetStatVarMatchRequest {
+  // The property value pair. Ex
+  // {gender: Male}, {statType: measuredValue}
+  // To improve this, can explicitly specify "statType", "measuredProperty",
+  // and constraint properties. Then the match could have must-match terms and
+  // optional match terms that have weights. That would greatly reduce the
+  // number of returned match stat vars.
+  map<string, string> property_value = 1;
+}
```

Comment:
I would probably add a `k` parameter to limit the number of returned matches. I don't think we are interested in downloading the full schema at every request.